src/main/java/views/TeamEditorPane.java

```diff
@@ -0,0 +1,4 @@
+package views;
+
+public class TeamEditorPane {
+}
```

Comment:
[OVERALL] Great job on the implementation, nicely chosen data structures and very nice encapsulation/separation !